x/ibc/testing/types.go

```diff
@@ -1,16 +1,59 @@
 package testing
 
+import (
+	"strconv"
+)
+
+var (
+	ChannelIDPrefix = "channelid"
+)
+
 // TestConnections is a testing helper struct to keep track of the connectionID, source clientID,
 // and counterparty clientID used in creating and interacting with a connection.
 type TestConnection struct {
 	ID                   string
 	ClientID             string
 	CounterpartyClientID string
+	Channels             []TestChannel
+}
+
+// AddTestChannel appends a new TestChannel which contains references to the port and channel ID
+// used for channel creation and interaction. The channel id and port id format:
+// channelid<index>
+// portid<index>
+func (conn *TestConnection) AddTestChannel() TestChannel {
+	channel := conn.NextTestChannel()
+	conn.Channels = append(conn.Channels, channel)
+	return channel
+}
+
+// NextTestChannel returns the next test channel to be created on this connection
+func (conn *TestConnection) NextTestChannel() TestChannel {
```

Comment:
```suggestion
func (conn *TestConnection) NewTestChannel() TestChannel {
```

Reply:
Hmm, so my issue with this naming is calling `NewTestChannel` would result in the same test channel being returned